scripts/manage_study.py

```diff
@@ -112,23 +117,37 @@ def manage_call_return(call_return, verbose=False):
 
 
 def succeeded(ret):
-    """Whether request succeeded
-    """
+    """Whether request succeeded"""
     # 2xx, e.g. 200 or 204, means success
-    return str(ret['response'].status_code)[0] == '2'
+    return str(ret["response"].status_code)[0] == "2"
 
 
 def login(parsed_args, manager=None, dry_run=False, api_base=None, verbose=False):
-    '''
+    """
     Login to authorize credentials.
 
     :param manager: API Manager
     :param dry_run: If true, will do a dry run with no actual execution of functionality.
     :return:
-    '''
+    """
     if manager is None:
         manager = scp_api.SCPAPIManager(verbose=verbose)
-        manager.login(token=parsed_args.token, dry_run=dry_run, api_base=api_base)
+        if parsed_args.user_agent:
+            ingest_pkg_version = pkg_resources.get_distribution(
+                "scp-ingest-pipeline"
+            ).version
+            portal_pkg_version = pkg_resources.get_distribution(
+                "single_cell_portal"
+            ).version
+            user_agent = f"single-cell-portal/{portal_pkg_version} (manage-study) scp-ingest-pipeline/{ingest_pkg_version} (ingest_pipeline.py)"
```

Comment:
Nit: extracting this into a function (e.g. `user_agent = get_user_agent()`) would keep `login` easy to digest.

```suggestion
            ingest_pkg_version = pkg_resources.get_distribution(
                "scp-ingest-pipeline"
            ).version
            portal_pkg_version = pkg_resources.get_distribution(
                "single_cell_portal"
            ).version
            user_agent = f"single-cell-portal/{portal_pkg_version} (manage-study) scp-ingest-pipeline/{ingest_pkg_version} (ingest_pipeline.py)"
```

Reply:
Great suggestion. Implemented in `78bc263`. Also realized obtaining the package version could fail if user decided to run manage_study.py so added some error handling for those situations.